apps/authoring/src/main/models/projects/model-project.mock.ts

```diff
@@ -2,6 +2,8 @@ import { ProjectData } from './model-projects.types';
 
 export const data: ProjectData = {
   name: 'Untitled Project',
+  description: '',
+  authors: '',
```

Comment:
As we're using a global state with controlled components, the properties must be initialized to avoid a common React warning in the console: `Warning: A component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa)...`